### DISCUSSION TITLE
The ':' symbol instead of '_' between date and time when Syncoid replication

### DISCUSSION THREAD

**Discussion by masterwishx** [jimsalterjrs/sanoid#836] [Q&A] [answered]
using unRAID6-Sainoid plugin for Unraid that use latest version of Sanoid
There is ':' symbol instead of '_' between date and time

When replication by Syncoid :
`system_cache_ssd/appdata@syncoid_MYServer_2023-07-25:20:00:14-GMT03:00`
instead of :  
`system_cache_ssd/appdata@syncoid_MYServer_2023-07-25_20:00:14-GMT03:00`

**Comment by numerfolt** [comment, marked as answer]
That's because syncoid has a different time parsing function than sanoid.
You can see [here](https://github.com/jimsalterjrs/sanoid/blob/dbcaeef1ac55bd3dd929b86a8f6082fef16f02b1/syncoid#L2200), that syncoid uses only `:` and no `_`

**Reply by masterwishx** [reply]
> You can see [here](https://github.com/jimsalterjrs/sanoid/blob/dbcaeef1ac55bd3dd929b86a8f6082fef16f02b1/syncoid#L2200), that syncoid uses only `:` and no `_`

Maybe should be fixed then?

**Reply by numerfolt** [reply]
I dont know whether it is intended or not, but nevertheless you could just open an issue about it?
